uefi/section_test.go

```diff
@@ -12,7 +12,7 @@ var (
 	emptySec     = make([]byte, 0)                                                     // Empty section
 	tinySec      = []byte{4, 0, 0, byte(SectionTypeRaw)}                               // Section header with no data
 	wrongSizeSec = append([]byte{40, 0, 0, byte(SectionTypeRaw)}, make([]byte, 20)...) // Section with a size mismatch
-	smallSec     = append([]byte{24, 0, 0, byte(SectionTypeRaw)}, make([]byte, 20)...) // 24 byte Section
+	smallSec     = append([]byte{22, 0, 0, byte(SectionTypeRaw)}, make([]byte, 18)...) // 24 byte Section
```

Comment:
The comment still says "24 byte Section".

Reply:
Done